### DISCUSSION TITLE
Editorconfig questions

### DISCUSSION THREAD

**Discussion by Bykiev** [AdamWhiteHat/BigDecimal#38] [Discussions]
Hey guys, where did you get this editorconfig rules? I didn't see such rules for C# anywhere, can we move to the default Visual Studio rules?

**Comment by Protiguous** [comment]
I added the `.editorconfig` files to the solution and to the projects so we can all have a standard for formatting, types, and whatnot.
Visual Studio, Resharper, and I believe Rider and VSCode, will all follow the settings specified in the .editorconfig files.

If @AdamWhiteHat makes a change to the `.editorconfig` files, then we can all pull down the new settings.

**Reply by AdamWhiteHat** [reply]
In the main branch, I removed extrainous .editor config files (there was one in the Tests project folder, as well as the helpers folder, and they both did not match the .editor config that was was in the solution's root directory) and ensured the .editorconfig in the solution file contained visual studio defaults.

**Reply by Protiguous** [reply]
I added the `.editorconfig` files there on purpose, so we can have different rules for tests, benchmarking, etc.

The “main” `.editorconfig` would just be the defaults for any project that didn't have one. That's also why it has the "root=true" file line.
